src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/GatewayIPAddressInformationCollection.cs

```diff
@@ -78,5 +78,18 @@ public virtual void Clear()
         {
             throw new NotSupportedException(SR.net_collection_readonly);
         }
+
+        internal bool Contains(IPAddress ipAddress)
```

Comment:
I can't find any reference of this. Is this dead code is it just VS not being able to discover the references? (I might have missed it during a text search.)